ironfish-cli/src/commands/service/bridge/deposit.ts

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Asset } from '@ironfish/rust-nodejs'
+import { WebApi } from '@ironfish/sdk'
+import { Flags } from '@oclif/core'
+import { IronfishCommand } from '../../../command'
+
+export class Deposit extends IronfishCommand {
+  static description = `Deposit coins to the bridge`
+
+  static flags = {
+    endpoint: Flags.string({
+      char: 'e',
+      description: 'API host to sync to',
+      parse: (input: string) => Promise.resolve(input.trim()),
+      env: 'IRONFISH_API_HOST',
+    }),
+    token: Flags.string({
+      char: 't',
+      description: 'API token to authenticate with',
+      parse: (input: string) => Promise.resolve(input.trim()),
+      env: 'IRONFISH_API_TOKEN',
+    }),
+    assetId: Flags.string({
+      char: 'i',
+      description: 'The identifier for the asset to deposit',
+    }),
+    source: Flags.string({
```

Comment:
should this be account? You will construct transaction in here too I am guessing?

Reply:
Updated to take `account` and to send the transaction!